lib/slax/commands/github_commands.ex

```diff
@@ -70,10 +81,12 @@ defmodule Slax.Commands.GithubCommands do
         results
 
       {:error, message} ->
-        Map.update(results, :errors, %{}, fn x -> Map.put(x, :resuseable_stories, message) end)
+        Map.update(results, :errors, %{}, fn x -> Map.put(x, :reusable_stories, message) end)
     end
   end
 
+  def create_reusable_stories(results, _, _, _, _), do: results
```

Comment:
The default case when no `github_repo` from a previous step exist is now to just return the results map